gcp/README.md

```diff
@@ -0,0 +1,100 @@
+# project_k8ssandra
+
+## Standerdization 
```

Comment:
Love that you have standards - please move this information to a DEVELOPER.md file in the root directory of the project